networks/cardano/tokens/e2087044f1a5028aec705eb05609370754d64587adbdc27eeb33005854414e47.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "tangent",
+    "decimals": 6,
+    "mldId": "cardano-TANG-e2087044f1a5028aec705eb05609370754d64587adbdc27eeb33005854414e47",
+    "orgName": "Tangent Protocol",
+    "orgUrl": "https://tangent.art/",
+    "tokenAddress": "e2087044f1a5028aec705eb05609370754d64587adbdc27eeb33005854414e47",
```

Comment:
this token token is deleted from cardano token registry
https://github.com/cardano-foundation/cardano-token-registry/pull/2753/files